libs/design/src/core/prefix-suffix/prefixable/prefixable.ts

```diff
@@ -0,0 +1,23 @@
+import { ContentChild } from '@angular/core';
+import { DaffPrefixDirective } from '../prefix.directive';
+import { Constructor } from '../../constructor';
+
+
+/**
+ * A mixin for giving a form control component a prefixed symbol.
```

Comment:
This comment doesn't make sense as currently defined.

Reply:
Updated in ff3c405